pkg/auth/tokenparser_test.go

```diff
@@ -8,67 +8,77 @@ import (
 	"github.com/codeready-toolchain/registration-service/pkg/auth"
 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
 	"github.com/codeready-toolchain/registration-service/test"
+	authsupport "github.com/codeready-toolchain/toolchain-common/pkg/test/auth"
 
 	"github.com/dgrijalva/jwt-go"
 	uuid "github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestTokenParser(t *testing.T) {
+type TestTokenParserSuite struct {
+	test.UnitTestSuite
+}
+
+func TestRunTokenParserSuite(t *testing.T) {
+	suite.Run(t, &TestTokenParserSuite{test.UnitTestSuite{}})
+}
+
+func (s *TestTokenParserSuite) TestTokenParser() {
 	configRegistry := configuration.CreateEmptyRegistry()
 
 	// create test keys
-	tokengenerator := test.NewTokenManager()
+	tokengenerator := authsupport.NewTokenManager()
 	kid0 := uuid.NewV4().String()
 	_, err := tokengenerator.AddPrivateKey(kid0)
-	require.NoError(t, err)
+	require.NoError(s.T(), err)
```

Comment:
actually, you don't have to use `s.T()` instead of `t` anymore, since https://github.com/stretchr/testify/commit/865fb2c8f5af9ccbaaadfd20d18236358dccaaff

Reply:
Do you mind if we make this change in another PR? I can create one and assign to to myself - was can make the change for all the tests in one sweep in that case. Currently all the tests use s.T().

Reply:
ok, let's proceed like that, then